Formula/kontena-cli.rb

```diff
@@ -0,0 +1,68 @@
+class KontenaCli < Formula
+  desc "Command-line client for Kontena container & microservices platform"
+  homepage "https://kontena.io/"
+  url "https://github.com/kontena/kontena.git", :tag => "v1.3.4"
+  head "https://github.com/kontena/kontena.git"
```

Comment:
would leave out head since the vendored gem dependencies will be a moving target

Reply:
Can livecheck keep the dependencies up to date or will that then require a new PR for all new versions?

Reply:
the resource blocks will have to be manually maintained.